metaswitch/common/pdlogs.py

```diff
@@ -60,8 +60,9 @@ def log(self, **kwargs):
     desc="The connection to Cassandra has been lost.",
     cause="The connection to Cassandra has been lost.",
     effect="The node can no long offer services that rely on Cassandra.",
```

Comment:
"The node can...".   What context is a PD log raised in?   Is it in the context of a node, or in the context of a process?   Does it make sense to talk about "the node" not being able to offer services that rely on Cassandra?   Presumably if there are two processes on a node that rely on Cassandra then either or both may independently make this log?